cubids/constants.py

```diff
@@ -1,7 +1,14 @@
 """Constants for CuBIDS."""
+
+# Names of identifier variables.
+# Used to place KeyGroup and ParamGroup at the beginning of a dataframe,
+# but both are hardcoded in the relevant function.
 ID_VARS = set(["KeyGroup", "ParamGroup", "FilePath"])
+# Entities that should not be used to group parameter sets
 NON_KEY_ENTITIES = set(["subject", "session", "extension"])
-# Multi-dimensional keys SliceTiming
+# Multi-dimensional keys SliceTiming  XXX: what is this line about?
```

Comment:
This was going to be a list of fields that are array or list type. They would need special consideration when deciding whether they're a variant or not.

Reply:
Ah, so it's a relic from an old attempt at #281?